src/askscript/__tests__/90-misc/scope-02.test.ts

```diff
@@ -0,0 +1,7 @@
+import program from './scope-02.ask';
```

Comment:
Why not have one test file for `let` and add all the possibilities there explicitly? 
It seems to me it is harder to maintain and much harder to understand the tests as they are now.
All in all, there are 2-3 scenarios per core functionality and it would be much easier to understand and follow if it were set in one file.
Something like:
```
import simpleAssign from './basics17-assign.ask';
import scopedAssign from './basics17-assign-scoped.ask';
import emptyAssign from './basics17-assign-empty.ask';

test('assign value for declared variable', async () => {
  await expect(await simpleAssign()).toEqual(2);
});

test('assign value inside scope for globally declared variable', async () => {
  await expect(await scopedAssign()).toEqual(2);
});

test('throw an error when assign to undeclared variable', async () => {
  await expect(emptyAssign()).rejects.toThrow();
});
```
@mhagmajer @czerwinskilukasz1 
What do you think?

Reply:
This is important and general subject. Let's take it into another issue. I also think that testing is very important and we should reconsider our approach here.

Reply:
https://github.com/xFAANG/askql/issues/249